app/Core/Auth.php

```diff
@@ -3,7 +3,7 @@
 namespace Application\Core\Auth;
 
 use Application\Core\Database\DatabaseConnexion\DatabaseConnexion;
-use Application\Repositories\User\UsersRepository;
+use Application\Repositories\UserRepository\UserRepository as UserRepository;
```

Comment:
C'est étrange normalement
```suggestion
use Application\Repositories\UserRepository\UserRepository;
```
ca devrait être bon